test/tpu/xla_test_job.yaml

```diff
@@ -44,7 +44,8 @@ spec:
       python3 pytorch/xla/test/pjrt/test_experimental_pjrt_tpu.py
       python3 pytorch/xla/test/spmd/test_xla_sharding.py
       python3 pytorch/xla/test/spmd/test_xla_virtual_device.py
-      python3 pytorch/xla/test/spmd/test_train_spmd_linear_model.py
+      XLA_USE_SPMD=1 python3 pytorch/xla/test/spmd/test_train_spmd_linear_model.py --sharding batch
+      XLA_USE_SPMD=1 python3 pytorch/xla/test/spmd/test_inference_spmd_dynamo_imagenet.py  --fake_data --use_dynamo --sharding batch
```

Comment:
awesome, thank you!

Reply:
How long will this test take? It is inference so I guess it won't be too bad but I am trying to prevent us from adding too many burden to the already very long CI.